snippets/capabilities-table.adoc

```diff
@@ -8,16 +8,16 @@ The following table describes the `baselineCapabilitySet` values.
 |Value|Description
 
 |`vCurrent`
-|Specify when you want to automatically add new capabilities as they become recommended.
+|Specify when you want to automatically add new capabilities as they become available.
 
 |`v4.11`
-|Specify when you want the capabilities recommended in {product-title} 4.11 and not automatically enable capabilities, which might be introduced in later versions. The capabilities recommended in {product-title} 4.11 are `baremetal`, `marketplace`, and `openshift-samples`.
+|Specify this option when you want the default capabilities for {product-title} 4.11 to be enabled. By specifying `4.11` in your `install-config.yaml` file, capabilities that are introduced in newer versions of {product-title} are not enabled. The default capabilities in {product-title} 4.11 are `baremetal`, `marketplace`, and `openshift-samples`.
```

Comment:
openshift/api#1554 makes this a bit sticky, by including `MachineAPI` in all of these in 4.14+.  Not a particularly impactful change for clusters updating from 4.13 and earlier, since the machine-API component was enabled there, and because [capabilities cannot be uninstalled][1], they'll be implicitly enabled in those clusters, even without the use explicitly asking for them.

But it is a shift for folks using `v4.11` and similar in install-configs once they start installing 4.14+ clusters.  This is mitigated by folks needing to check for new capabilities before recycling an older install-config in a new 4.y to see if they want to opt in or out of any new capabilities, although we may not doc that explicitly today.

But it does make it a bit fiddly to list the defaults here, because they shift from your current listing in 4.13 and earlier to that-listing-plus-`MachineAPI` in 4.14 and later.

[1]: https://github.com/openshift/enhancements/blob/6cebc13f0672c601ebfae669ea4fc8ca632721b5/enhancements/installer/component-selection.md#capabilities-cannot-be-uninstalled

Reply:
Also:

> By specifying `4.11`...

It's [`v4.11`, with the `v` prefix][1], and similarly for the other values.  Also, it's not just install-config, you're also free to adjust these post-install in ClusterVersion's `spec.capabilities`, with the caveat that [capabilities cannot be uninstalled][2].

[1]: https://github.com/openshift/api/blame/1b50bb45038b24512b2ec18dc19de58269f3cf8c/config/v1/types_cluster_version.go#L356-L392
[2]:https://github.com/openshift/enhancements/blob/6cebc13f0672c601ebfae669ea4fc8ca632721b5/enhancements/installer/component-selection.md#capabilities-cannot-be-uninstalled

Reply:
@wking Should we remove " in your `install-config.yaml` file" and simply make it the following?

|Specify this option when you want the default capabilities for {product-title} 4.11 to be enabled. By specifying `v4.11`, capabilities that are introduced in newer versions of {product-title} are not enabled. The default capabilities in {product-title} 4.11 are `baremetal`, `marketplace`, and `openshift-samples`.

This allows me having to mention CVO's `spec.capabilities` field entirely, in this section.